generator/internal/genclient/translator/protobuf/protobuf_test.go

```diff
@@ -28,59 +30,7 @@ import (
 )
 
 func TestScalar(t *testing.T) {
-	contents := `
```

Comment:
Could you do this in a refactoring PR? It is harder to grok new code and cleanups in the same change.

Reply:
yeah, that is a good idea: https://github.com/googleapis/google-cloud-rust/pull/101